torch/csrc/autograd/python_variable.cpp

```diff
@@ -289,54 +389,11 @@ static int THPVariable_clear(THPVariable* self) {
       }
     }
   }
+  TORCH_INTERNAL_ASSERT(!isResurrectable((THPVariable*)self));
```

Comment:
Does this check still makes sense here? Why do we test this again even after we did it at the beginning of the function?

Reply:
See the failures here: https://github.com/pytorch/pytorch/pull/75933/commits/fd06e184863737cd2359b59767bb5b4eeb804c0f